build_travis.sh

```diff
@@ -61,21 +61,25 @@ function main() {
 
     # create a new pull request to master, if pushed to staging
     if [[ "${branch}" == 'staging' && $(ls *.pdf) ]]; then
+        git config 'user.name' 'noto-buildbot'
+        git config 'user.email' 'noto-buildbot@google.com'
         git checkout artifact-branch
         #TODO could put results in a directory just for this change, to simplify
         # simultaneous reviews. would also have to to remove it afterwards.
         mv *.pdf artifacts
         git add artifacts
         git commit -m 'Review commit' --amend
-        git push --force origin artifact-branch
-        json='{
-            "title": "Review request",
-            "body": "Review build results at
-                https://github.com/googlei18n/noto-source/tree/artifacts-branch/artifacts",
-            "head": "staging",
-            "base": "master"
-        }'
-        curl -u "noto-buildbot:${noto_buildbot_token}" -d "${json}"\
+        git_url='github.com/googlei18n/noto-source'
+        credentials="noto-buildbot:${noto_buildbot_token}"
+        git push --force "https://${credentials}@${git_url}.git" artifact-branch
```

Comment:
The problem is, if we build each source 20 times and the artefacts produced are all different, because for example pipeline changed, then the repo will be containing all the 20 different artefacts, forever.  Whereas the sources themselves don't change often.

Anyway, something to keep an eye on.  We can always prune the repo later.